Buildings/Experimental/DHC/Examples/Cooling/DirectUncontrolledETS.mo

```diff
@@ -0,0 +1,184 @@
+      color={255,204,51},
+      thickness=0.5));
+  connect(on.y, pla.on) annotation (Line(points={{-59,-10},{-50,-10},{-50,-2.6},
+          {-40.7333,-2.6}}, color={255,0,255}));
+  connect(TCHWSupSet.y, pla.TCHWSupSet) annotation (Line(points={{-59,-50},{-48,
+          -50},{-48,-4.73333},{-40.6667,-4.73333}}, color={0,0,127}));
+  connect(senRelPre.p_rel, pla.dpMea) annotation (Line(points={{-17,-36},{-46,
+          -36},{-46,-6.73333},{-40.6667,-6.73333}},
+                                               color={0,0,127}));
+  connect(dis.port_bDisSup, dis.port_aDisRet) annotation (Line(points={{60,-10},
+          {80,-10},{80,-16},{60,-16}}, color={0,127,255}));
+  connect(pla.port_aSerCoo, dis.port_bDisRet) annotation (Line(points={{-40,
+          -11.3333},{-44,-11.3333},{-44,-60},{8,-60},{8,-16},{20,-16}}, color={
+          0,127,255}));
+  connect(dis.port_aDisSup, pla.port_bSerCoo) annotation (Line(points={{20,-10},
+          {0,-10},{0,-11.3333},{-20,-11.3333}}, color={0,127,255}));
+  connect(senRelPre.port_b, dis.port_bDisRet) annotation (Line(points={{-8,-46},
+          {-8,-60},{8,-60},{8,-16},{20,-16}}, color={0,127,255}));
+  connect(senRelPre.port_a, pla.port_bSerCoo) annotation (Line(points={{-8,-26},
+          {-8,-11.3333},{-20,-11.3333}}, color={0,127,255}));
+  connect(dis.ports_bCon, buiETS.port_aSerCoo)
+    annotation (Line(points={{28,0},{0,0},{0,42},{30,42}}, color={0,127,255}));
+  connect(buiETS.port_bSerCoo, dis.ports_aCon) annotation (Line(points={{50,42},
+          {80,42},{80,0},{52,0}}, color={0,127,255}));
+    annotation (Dialog(group="Network"),
+    Icon(
+      coordinateSystem(
+        preserveAspectRatio=false)),
+    Diagram(
+      coordinateSystem(
+      preserveAspectRatio=false)),
+    __Dymola_Commands(
+      file="modelica://Buildings/Resources/Scripts/Dymola/Experimental/DHC/Examples/Cooling/DistrictCooling.mos" "Simulate and plot"),
+    experiment(
+      StartTime=12960000,
+      StopTime=13564800),
+    Documentation(info="<html>
+<p>This model illustrates an example of integral district cooling system, 
+consisted by a cooling plant of parallel electric chillers 
+<a href=\"modelica://Buildings/Experimental/DHC/Plants/Cooling/ElectricChillerParallel.mo\">
+Buildings.Experimental.DHC.Plants.Cooling.ElectricChillerParallel</a>, 
+a two-pipe distribution network <a href=\"modelica://Buildings/Experimental/DHC/Networks/Distribution2Pipe.mo\">
+Buildings.Experimental.DHC.Networks.Distribution2Pipe</a>, 
+and a time series building load connected to a direct uncontrolled ETS for cooling 
+<a href=\"modelica://Buildings/Experimental/DHC/Loads/Cooling/BuildingTimeSeriesWithETS.mo\">
+Buildings.Experimental.DHC.Loads.Cooling.BuildingTimeSeriesWithETS</a>, as illustrated in the schematic below.</p>
+<p align=\"center\"><img src=\"modelica://Buildings/Resources/Images/Experimental/DHC/Examples/Cooling/DirectUncontrolledETS.png\"/></p>
```

Comment:
<img> lacks "alt" attribute. It should be
```
<img src=\"modelica://Buildings/Resources/Images/Experimental/DHC/Examples/Cooling/DirectUncontrolledETS.png\" alt=\"xxxxx\" />
```

Reply:
Thanks for reminding, already added "alt" attribute.